lxljs/vocab.js

```diff
@@ -453,22 +450,16 @@ export function getProperties(classId, vocabClasses, vocabProperties, context) {
   if (termObj.allowedProperties) {
     return termObj.allowedProperties;
   }
-  vocabProperties.forEach((prop) => {
-    const domainList = getDomainList(prop, vocabProperties, context);
-    let domainListWithSubClasses = [];
-    for (let i = 0; i < domainList.length; i++) {
-      domainListWithSubClasses = domainListWithSubClasses.concat(
-        getSubClassChain(domainList[i], vocabClasses, context),
-      );
-    }
-    for (const domain of domainListWithSubClasses) {
-      if (domain === classId) {
-        props.push(prop);
+
+  const allowedProperties = Array.from(vocabProperties.values())
+    .filter((prop) => {
+      if (getPropertiesList(prop, vocabProperties, context).includes(classId)) {
```

Comment:
This appears to result in the set of "allowed properties for this class" including properties who have this class as their range (which would be backwards)? Am I missing something?

Reply:
> 

<img width="1624" alt="Skärmavbild 2023-05-24 kl  09 03 26" src="https://github.com/libris/lxlviewer/assets/10220360/17bd7432-e8a4-4721-a95a-cc583ebe335e">
Should the properties listed under _Occurs in_ in the screenshot be separated into two separate table rows (one for "Förekommande egenskaper" and one for "Förekommer i egenskaper")? Would that be correct?

Reply:
Yes. the formal view for class Person would be: "in domain of": [dates and names ...], and "in range of": [subject, hasProminentFamilyMember]. The Swedish labels are not set in stone; we need to agree on formal names here (ping @klngwll @andand79 ).